### PR TITLE
orchestration api added

### PR DESCRIPTION
**Pull request by rakeshSgr**

### CODE REVIEW

File: elevate-user/controllers/user.js [hunk 1]
```diff
@@ -23,6 +23,23 @@ const entityTypeRead = async (req, res, responses) => {
 	})
 }
 
+const userDetails = async (req, res, responses) => {
+	const selectedConfig = routeConfigs.routes.find((obj) => obj.sourceRoute === req.sourceRoute)
+	
+	const parameterisedRoute = selectedConfig.targetRoute.path;
+	  
+	let headers = {
+		  'internal_access_token': req.headers['internal_access_token'],
```
Comment (aks30) [inline elevate-user/controllers/user.js:32]: @rakeshSgr Please remove this

File: elevate-user/constants/routes.js [hunk 2]
```diff
@@ -960,7 +960,29 @@ module.exports = {
 				path: '/user/v1/admin/triggerViewRebuildInternal',
 				type: 'GET',
 			},
-		}
+		},
+		{
+			sourceRoute: '/interface/v1/mentors/details/:id',
```
Comment (aks30) [inline elevate-user/constants/routes.js:965]: @rakeshSgr check if this is needed.

File: elevate-mentoring/utils/requester.js [hunk 3]
```diff
@@ -89,11 +90,30 @@ const patch = async (baseUrl, route, requestBody, headers) => {
...
 }
+const get = (baseUrl, route, headers, requestBody = {}) => {
+	const url = baseUrl + route
+	const options = {
+		headers,
+		data: requestBody 
+	};
+
+	return axios
+		.post(url, requestBody, options) // Use POST to send body data
```
Comment (aks30) [inline elevate-mentoring/utils/requester.js:101]: @rakeshSgr this should be GET

File: elevate-mentoring/controllers/mentoring.js [hunk 4]
```diff
@@ -31,11 +31,50 @@ const rolePermissions = async (req, res, responses) => {
...
+	  }
+
+	  console.log("response ============ headers  ",headers);
+	
+	 let response = await requesters.get(req.baseUrl, parameterisedRoute,headers,{})
+	  return res.json(response)
+
+}
+
+const mentoringProfile = async (req, res, responses) => {
```
Comment (aks30) [inline elevate-mentoring/controllers/mentoring.js:55]: @rakeshSgr check if this is needed, can it be pass through?

**Review by aks30 — CHANGES_REQUESTED** [review]
Reviewed on 11 Nov

File: elevate-user/constants/routes.js (hunk 2 above)
Comment (rakeshSgr) [reply elevate-user/constants/routes.js:965]: yes it required

File: elevate-mentoring/controllers/mentoring.js (hunk 4 above)
Comment (rakeshSgr) [reply elevate-mentoring/controllers/mentoring.js:55]: yes its required.

**Pull request merged**